app/src/main/res/layout-sw600dp-land/onboarding_slide_final.xml

```diff
@@ -42,8 +42,7 @@
         android:fontFamily="sans-serif-medium"
         android:gravity="center_horizontal"
         android:text="@string/onboarding_slide_3_title"
```

Comment:
Apply all the changes to this file which I have mentioned previously.

Reply:
Made the change!